### PR TITLE
Make the network.services field more clear

### PR DESCRIPTION
**Pull request by ptnapoleon**
## What does this PR do?

- [ ] Adds new functionality
- [ ] Alters existing functionality
- [ ] Fixes a bug
- [ ] Improves documentation or testing

Please briefly describe your changes as well as the motivation behind them:
- Update all docs and examples to explain that network.services is only for the same cluster

## Code Quality Checklist

- [ ] The documentation is up to date.
- [ ] My code is sufficiently commented and passes continuous integration checks.
- [ ] I have signed my commit (see [Contributing Docs](../CONTRIBUTING.md)).

## Testing

- [ ] I leveraged continuous integration testing
    - [ ] by depending on existing `unit` tests or `end-to-end` tests.
    - [ ] by adding new `unit` tests or `end-to-end` tests.
- [ ] I manually tested the following steps:
    - `x`
    - [ ] locally.
    - [ ] as a canary deployment to a cluster.

### CODE REVIEW

**Review by aymericDD — APPROVED** [review]
👌

File: examples/complete.yaml [hunk 1]
```diff
@@ -63,7 +63,7 @@ spec:
         protocol: tcp # optional, protocol to filter on (can be tcp or udp, defaults to both)
         flow: ingress # optional, flow direction (egress: outgoing traffic, ingress: incoming traffic, defaults to egress)
         connState: new # optional, connection state (new: new connections, est: established connections, defaults to all states)
-    services: # optional, list of destination Kubernetes services to filter on
+    services: # optional, list of destination Kubernetes services to filter on. these must be in the same kubernetes cluster
```
Comment (clairecng) [inline examples/complete.yaml:66]:
```suggestion
    services: # optional, list of destination Kubernetes services to filter on. These must be in the same kubernetes cluster
```

**Review by clairecng — APPROVED** [review]
nit

**Pull request merged**